CharlieBackend.Business/Services/MentorService.cs

```diff
@@ -173,5 +173,25 @@ public async Task<MentorDto> GetMentorByIdAsync(long mentorId)
 
             return mentor?.AccountId;
         }
+
+        public async Task<Result<MentorDto>> DisableMentorAsync(long mentorId)
+        {
+            var accountId = await GetAccountId(mentorId);
+
+            if (accountId == null)
+            {
+                return Result<MentorDto>.GetError(ErrorCode.NotFound, "Unknown mentor id.");
+            }
+
+            var mentor = await GetMentorByAccountIdAsync((long)accountId);
+            var isActive = await _accountService.DisableAccountAsync((long)accountId);
+
+            if (!isActive)
```

Comment:
May we put this check in "isActive" in "If"?

Reply:
May